### PR TITLE
Chore/clippy lints

### PR DESCRIPTION
**Pull request by distributedstatemachine**

### CODE REVIEW

File: pallets/commitments/src/lib.rs [hunk 1]
```diff
@@ -295,11 +301,7 @@ where
         _len: usize,
         _result: &DispatchResult,
     ) -> Result<(), TransactionValidityError> {
-        if let Some((call_type, _transaction_fee, _who)) = maybe_pre {
-            match call_type {
-                _ => (),
-            }
-        }
+        if let Some((call_type, _transaction_fee, _who)) = maybe_pre {}
```
Comment (open-junius) [inline pallets/commitments/src/lib.rs]: can just remove it.

File: pallets/subtensor/tests/senate.rs [hunk 2]
```diff
@@ -145,7 +145,7 @@ fn test_senate_vote_works() {
         ));
         // Check if balance has  decreased to pay for the burn.
         assert_eq!(
-            SubtensorModule::get_coldkey_balance(&coldkey_account_id) as u64,
+            { SubtensorModule::get_coldkey_balance(&coldkey_account_id) },
```
Comment (open-junius) [inline pallets/subtensor/tests/senate.rs]: {} can be removed

File: pallets/subtensor/tests/senate.rs [hunk 3]
```diff
@@ -254,7 +254,7 @@ fn test_senate_vote_not_member() {
...
```
Comment (open-junius) [inline pallets/subtensor/tests/senate.rs]: the same as line 148

File: pallets/subtensor/tests/senate.rs [hunk 4]
```diff
@@ -315,7 +315,7 @@ fn test_senate_leave_works() {
...
```
Comment (open-junius) [inline pallets/subtensor/tests/senate.rs]: the same as line 148

**Review by open-junius — APPROVED** [review]

**Review by sam0x17 — APPROVED** [review]
looks good once the conflicts are fixed

**Pull request merged**